generators/app/index.js

```diff
@@ -40,52 +41,58 @@ module.exports = yeoman.Base.extend({
 
       gems: _(this.gems).concat(
         "gem 'pg', '~> 0.18'",
-        "gem 'puma'",
-        "gem 'rack-cors'"
+        "gem 'puma', '~> 3.0'"
       ).sortBy().join('\n'),
 
       generatorVersion: this.fs.readJSON(require.resolve('../../package.json')).version,
 
-      test_gems: _(this.testGems).concat(
-        "gem 'byebug', platform: :mri"
-      ).sortBy().join('\n  ')
+      testGems: _(this.testGems).concat(
+        "gem 'byebug', platform: :mri",
+        "gem 'dotenv'"
+      ).sortBy().join('\n  '),
+
+      _: _ //lodash
     };
   },
 
   writing: function () {
-    var from = this.templatePath('.');
-    var to = this.destinationPath('.');
-
-    this.fs.copyTpl(from, to, _.merge(this.templateVars, { _: _ }));
-
-    // copyTpl seems to mess with binary files (which is understandable)
-    this.fs.copy(
-      this.templatePath('app/client/assets/foo-asset.png'),
-      this.destinationPath('app/client/assets/foo-asset.png')
-    );
-
-    // See https://github.com/npm/npm/issues/7252 for why this is needed
-    for (let file of fs.readdirSync(this.templatePath())) {
-      if (file.charAt(0) === '_') {
-        this.fs.move(
-          this.destinationPath(file),
-          this.destinationPath(`.${file.substr(1)}`)
-        );
-      }
+    var files = glob.sync('**', {
+      cwd: this.sourceRoot(),
+      dot: true
+    });
+
+    files.forEach(function (file) {
+      this.fs.copyTpl(
+        this.templatePath(file),
+        this.destinationPath(file),
+        this.templateVars
+      );
+    }.bind(this));
+
+    if (this.envFile) {
+      this.fs.write(this.destinationPath('shared/.env'), this.envFile);
+    } else {
+      this.fs.copyTpl(
+        this.templatePath('shared/.env'),
+        this.destinationPath('shared/.env'),
+        this.templateVars
+      );
     }
+
     this.config.save();
   },
 
   install: function() {
-    this.composeWith('prelude:setup', {}, { local: require.resolve('../setup') });
+    this.spawnCommand('chmod', ['+x', 'bin/setup'], { cwd: this.destinationPath() });
+    this.spawnCommand('chmod', ['+x', 'bin/start'], { cwd: this.destinationPath() });
```

Comment:
The node platform, from my recent experiences, is surprisingly well supported on Windows. Ruby is not, but it is better than years ago. Anyway I can't really see me developing on Windows right now, but some people do. Assuming Ruby is correctly installed and node is correctly installed, the generator as it is nowadays will work on Windows just ok. I test it before publishing any new version.